frontend/src/tabs/nfts/NFTItem.js

```diff
@@ -54,6 +54,12 @@ const NFTItem = (props) => {
   const posy = Math.floor(props.position / canvasConfig.canvas.width);
 
   const [minterText, setMinterText] = React.useState('');
+
+  function handleShare(){
+    const twitterShareUrl = `https://x.com/intent/post?text=${encodeURIComponent("Hey, check out my nft")}&url=${encodeURIComponent(props.image)}`
```

Comment:
Very cool!

Can you make it so the message is different if clicking `handleShare` depending on one of 3 cases:
1. The user minted and owns the NFT ( Check out this nft I minted )
2. The user owns but didn't mint the NFT ( Check out this NFT I got )
3. The owner neither owns nor minted the NFT ( Check out this NFT )